data/sql/updates/pending_db_characters/Homebind_Orientation.sql

```diff
@@ -1,4 +1,5 @@
 --
+DELETE FROM `updates` WHERE  `name`='Homebind_Orientation.sql';
```

Comment:
no need this

Reply:
Its needed to avoid the yellow error on start up as a missing update.